systemFlake.nix

```diff
@@ -149,9 +159,9 @@ let
             ];
           })
         ] ++ host.modules;
-        specialArgs = host.specialArgs;
+        inherit specialArgs;
```

Comment:
this was in failure mode before. we have to remain careful about the builder generalization.

Reply:
There was actually a reason for this: setting modulesPath on a non-nixos builder to a nixos path would break things.
I'll address this in #44

Reply:
Ok thanks! We have to be really careful about rhose kind of things. There are now +- 5 design dimenstion in three lines of code.

Reply:
Maybe we'd need to split them eventually apart completely and implement a generic builder and a specific nixos one on top.